lips/lip-1.mdx

```diff
@@ -203,7 +143,7 @@ All responses to a CommandRequestObject are in the form of a CommandResponseObje
 | _ObjectType    | str      | Y             | The fixed string `CommandResponseObject`. |
 | status         | str      | Y             | Either `success` or `failure`. |
 | error          | List of [OffChainErrorObject](#offchainerrorobject) | N | Details on errors when status == "failure" |
-| cid            | str      | N | The Command identifier to which this is a response. |
+| cid            | str      | N | The Command identifier to which this is a response. Should be a UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122). |
```

Comment:
Same comment here

Reply:
And if we do end up not needing these, we need to remove them from all examples as well